samples/Mvc.Server/Startup.cs

```diff
@@ -135,7 +135,7 @@ public class Startup {
                     var hasher = new PasswordHasher<Application>();
 
                     context.Applications.Add(new Application {
-                        Id = "myClient",
+                        ApplicationId = "myClient",
```

Comment:
Then maybe we rename Application entity to Client entity? Or do you have some intent of having different names?